docs/LogicalObjects.rst

```diff
@@ -40,7 +40,7 @@ otherwise they're pretty much just like they are in Objective-C.
 That said, they do interact with some other language features in interesting
 ways.
 
-For example, methods on value types have a this parameter.  Usually parameters
```

Comment:
This is correct as-is. It could use code voice for clarity but it is not necessary to be interpretable.